Assets/MixedRealityToolkit/_Core/Definitions/Devices/ControllerMappingLibrary.cs

```diff
@@ -17,213 +17,161 @@ public static class ControllerMappingLibrary
-            "OTOUCH_THUMBREST_LEFT_CONTROLLER",                 // 16 - THUMBREST_LEFT_CONTROLLER
-            "OTOUCH_THUMBREST_RIGHT_CONTROLLER"                 // 17 - THUMBREST_RIGHT_CONTROLLER
-        };
-
-        private static InputManagerAxis[] OculusTouchControllerAxisMappings =
-        {
-            new InputManagerAxis { Name = OculusTouchInputMappings[4], Dead = 0.001f, Sensitivity = 1, Invert = false, Type = InputManagerAxisType.JoystickAxis, Axis = 1 },
-            new InputManagerAxis { Name = OculusTouchInputMappings[5], Dead = 0.001f, Sensitivity = 1, Invert = false, Type = InputManagerAxisType.JoystickAxis, Axis = 2 },
-            new InputManagerAxis { Name = OculusTouchInputMappings[6], Dead = 0.001f, Sensitivity = 1, Invert = false, Type = InputManagerAxisType.JoystickAxis, Axis = 4 },
-            new InputManagerAxis { Name = OculusTouchInputMappings[7], Dead = 0.001f, Sensitivity = 1, Invert = false, Type = InputManagerAxisType.JoystickAxis, Axis = 5 },
-            new InputManagerAxis { Name = OculusTouchInputMappings[8], Dead = 0.001f, Sensitivity = 1, Invert = false, Type = InputManagerAxisType.JoystickAxis, Axis = 9 },
-            new InputManagerAxis { Name = OculusTouchInputMappings[9], Dead = 0.001f, Sensitivity = 1, Invert = false, Type = InputManagerAxisType.JoystickAxis, Axis = 10 },
-            new InputManagerAxis { Name = OculusTouchInputMappings[10], Dead = 0.001f, Sensitivity = 1, Invert = false, Type = InputManagerAxisType.JoystickAxis, Axis = 11 },
-            new InputManagerAxis { Name = OculusTouchInputMappings[11], Dead = 0.001f, Sensitivity = 1, Invert = false, Type = InputManagerAxisType.JoystickAxis, Axis = 12 },
-            new InputManagerAxis { Name = OculusTouchInputMappings[12], Dead = 0.001f, Sensitivity = 1, Invert = false, Type = InputManagerAxisType.JoystickAxis, Axis = 15 },
-            new InputManagerAxis { Name = OculusTouchInputMappings[13], Dead = 0.001f, Sensitivity = 1, Invert = false, Type = InputManagerAxisType.JoystickAxis, Axis = 16 },
-            new InputManagerAxis { Name = OculusTouchInputMappings[14], Dead = 0.001f, Sensitivity = 1, Invert = false, Type = InputManagerAxisType.JoystickAxis, Axis = 13 },
-            new InputManagerAxis { Name = OculusTouchInputMappings[15], Dead = 0.001f, Sensitivity = 1, Invert = false, Type = InputManagerAxisType.JoystickAxis, Axis = 14 },
-            new InputManagerAxis { Name = OculusTouchInputMappings[16], Dead = 0.001f, Sensitivity = 1, Invert = false, Type = InputManagerAxisType.JoystickAxis, Axis = 17 },
-            new InputManagerAxis { Name = OculusTouchInputMappings[17], Dead = 0.001f, Sensitivity = 1, Invert = false, Type = InputManagerAxisType.JoystickAxis, Axis = 18 }
-        };
+        /// <summary>
+        /// TOUCHPAD-THUMSTICK_LEFT_CONTROLLER_HORIZONTAL
+        /// </summary>
+        public const string MIXEDREALITY_OPENVR_AXIS1 = "MIXEDREALITY_OPENVR_AXIS1";
```

Comment:
What about controllers that use Axis1 and aren't part of OpenVR?

Reply:
Then they would be different controllers?
Also might have different tollerances...
SO, if the Xbox used Axis1, it may be configured differently and be different.  These particular axis settings are customised for the OpenVR spec.
We can review as we add more controllers later.

Reply:
We could always adjust the tolerance after we read the input.

Reply:
I think we only need to define a single set of axes for each.